src/android/AudioPlayer.java

```diff
@@ -532,6 +532,27 @@ public void setVolume(float volume) {
         }
     }
 
+     /**
+     * Set the rate for audio player
+     *
+     * @param rate
+     */
+
+    public void setRate(float rate) {
+        if (this.player != null) {
+              
+            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M) {
```

Comment:
I'm not sure if there is a guideline here, or what is the standard behavior in this cases, but should whe not send an error stats or something when the version does not meet the minimum requirement?